### DISCUSSION TITLE
Enhancement plugin goes to email.

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1174] [Q&A] [answered]
Just added the Enhancement plugin. Cool idea. I open a meeting and click the update button. It opens my email client. It was my understanding it would replace the map and give all the cool stuff to be able to change, which then would be emailed. I am not sure why I am not getting the enhancements.

What do you need from me? The site is test only. dougwade.com I hope to release it at the end of the month after the CO board approves it.

**Comment by joshreisner** [comment, marked as answer]
hi @dougie-wougie sorry the enhancement plugin only supports the "legacy" TSML appearance currently. looks like you're using TSML UI which is limited by being a frontend react app without access to the WP backend

**Comment by dougie-wougie** [comment]
Thank you. I am disappointed, but I understand. I love your work; keep making it strong.

**Comment by virtualtom** [comment]
@dougie-wougie check this discussion https://github.com/code4recovery/12-step-meeting-list/discussions/1216
I just did this on the buffaloaany.org site.  I created a form in WPForms and embedded it in a page so it has a URL, they you can update the functions.php of your sites theme to have the button open the form page instead of an email.  I like this better than the enhancement plugin because I don't want absolutely everyone to be presented everything that can be updated, I have a separate form for trusted servants to fill out for that.  I want this to be more limited feedback like the meeting didn't meet the last 2 times I was there, or this is a better description of how to get into the building.

**Reply by dougie-wougie** [reply]
Yes, I want to do that. I created the WPform and a page for it. Thank you for the sample. I got it working after doing one meeting update. Then all meeting by exporting/importing all the meetings
